### DISCUSSION TITLE
''Read only filesystem'' cannot place overlay

### DISCUSSION THREAD

**Discussion by hotChik** [phhusson/treble_experimentations#2503] [Q&A]
after building my overlay, i am trying to place it in /system/product/overlay/, but every time i do it i get the read only file system, i tried the mount commands in adb but they do not work,

phone is rog 5s cn edition, anyone knows what to do?

**Comment by phhusson** [comment]
you tried `adb root` ; `adb remount system` ; `adb push overlay.apk
/system/product/overlay`? can you show screenshots of your terminal when
you do that?

**Reply by hotChik** [reply]
Excuse me if this is simple or i have done an obvious mistake, but i have been trying to do this for 2 weeks on and off and really do not now what to do, here is my terminal screenshot

![image](https://user-images.githubusercontent.com/109736487/211162848-9ec35637-3838-48e2-b1c7-3f3ea7669f09.png)

**Reply by phhusson** [reply]
k that means you're not using a ROM meant for development. Use a ROM meant for development, like https://github.com/phhusson/treble_experimentations/releases/download/v416/system-squeak-arm64-ab-vanilla.img.xz
